gluoncv/model_zoo/yolo/yolo_target.py

```diff
@@ -116,8 +116,8 @@ def forward(self, img, xs, anchors, offsets, gt_boxes, gt_ids, gt_mixratio=None)
                     index = _offsets[nlayer] + loc_y * width + loc_x
                     center_targets[b, index, match, 0] = gtx / orig_width * width - loc_x  # tx
                     center_targets[b, index, match, 1] = gty / orig_height * height - loc_y  # ty
-                    scale_targets[b, index, match, 0] = np.log(gtw / np_anchors[match, 0])
-                    scale_targets[b, index, match, 1] = np.log(gth / np_anchors[match, 1])
+                    scale_targets[b, index, match, 0] = np.log(gtw / np_anchors[match, 0] + 1e-8)
```

Comment:
`max(gtw, 1e-8) / np_anchors[match, 0]` maybe better, and 1e-8 may still be too small (log(1e-8)=-18)

Reply:
yes, that is more straightway, i think set epsilon = ```1e-2``` maybe enough as ```gtw```, ```gth``` should ```> 1``` in regular case.

Reply:
since it's pixel level, it can be  `np.log(max(gtw, 1) / np_anchors[match, 0])`
@jianantian Can you make a change?

Reply:
ok